src/makeInfinityAdder.js

```diff
@@ -3,8 +3,24 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(currentNum) {
+  let currentNummount = currentNum || 0;
+
+  function sum(nextNum) {
+    if (nextNum !== undefined) {
+      currentNummount += nextNum;
+
+      return sum;
+    } else {
+      const result = currentNummount;
```

Comment:
The variable `currentNummount` is used here, which is a typo. It should be `currentNum` to correctly reference the accumulated value.